packages/x-data-grid/src/components/columnsPanel/ColumnsPanelTrigger.tsx

```diff
@@ -72,7 +72,6 @@ const ColumnsPanelTrigger = forwardRef<HTMLButtonElement, ColumnsPanelTriggerPro
       if (open) {
         event.stopPropagation();
       }
-      onPointerUp?.(event);
```

Comment:
Is this intended? Looks like onPointerUp from props will override `handlePointerUp`